docs/configuration/config-file-reference.md

```diff
@@ -1811,6 +1811,10 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -validation.enforce-metric-name
 [enforce_metric_name: <boolean> | default = true]
 
+# Per-user subring to shard metrics to ingesters. 0 is disabled.
+# CLI flag: -distributor.user-subring-size
```

Comment:
I'm wondering if we should prefix it with `-experimental.`. Thoughts?

Reply:
I think that's probably a good idea

Reply:
Done.